aepsych/utils.py

```diff
@@ -259,9 +259,10 @@ def _get_ax_parameters(config):
     for parname in fixed_parnames:
         try:
             try:
-                value = config.getfloat(parname, "value")
+                value: Union[float, str] = config.getfloat(str(parname), "value")
```

Comment:
This should always be a float.

Reply:
However, when the code reaches the following block:

```python
except ValueError:
    value = config.get(str(parname), "value")
```
The `value` could potentially be a string. Therefore, to handle this properly, we need to account for both possibilities. I'd be open to alternative approaches if there are better ways to resolve the mypy error related to this.